common/config/rush/browser-approved-packages.json

```diff
@@ -538,6 +538,10 @@
       "name": "cypress",
       "allowedCategories": [ "production", "tools" ]
     },
+    {
+      "name": "cypress-real-events",

```

Comment:
Why is this package needed? THe description mentions some very specific use cases :-/ https://github.com/dmtrKovalenko/cypress-real-events#why

Reply:
Thats because cypress not provide "hover" event, and you have to simulate by "mouseenter" and "mouse leave" and test like I did ("should show menu toggler button when mouse hovers over the cell (SEPARATE)") not make sense.

In KD this package is used too. Only limitation is that this package run just on the top Chrome. (Our case).